README.md

```diff
@@ -342,36 +342,42 @@ const NewBookForm = () => {
 
 ### Sequential API Execution
 
+Passing the 'isLoading' value from one useFetchye call to the 'defer' field of the next will prevent the second call from being made until the first has loaded.
+
+To ensure the second api call is properly formed, you should also check that the data you expect from the first call exists:
+
 ```jsx
 import React from 'react';
 import { useFetchye } from 'fetchye';
 
 const MyFavoriteBook = () => {
   const { isLoading: loadingProfile, data: profile } = useFetchye('http://example.com/api/profile');
-  const { isLoading: loadingBook, data: favoriteBook } = useFetchye(() => `http://example.com/api/books/${profile.body.favoriteBookId}`);
```

Comment:
this example relies on an error being thrown `Cannot read properties of undefined `. I think the defer example will be easier to understand. 

Might still be worth having an example of `useFetchye` receiving a function instead of a string

Reply:
Updated to include original style example